test/model_tests/chats/chat_message_test.dart

```diff
@@ -1,25 +0,0 @@
-import 'package:flutter_test/flutter_test.dart';
-import 'package:talawa/models/chats/chat_message.dart';
-import 'package:talawa/models/chats/chat_user.dart';
-
-final chatUserOne =
-    ChatUser(firstName: 'Ravidi', id: '1', image: "https://testimg.com");
-final chatUserTwo =
-    ChatUser(firstName: 'Sheikh', id: '2', image: "https://testimg.com");
-void main() {
-  group("Tests for ChatMessage.dart", () {
-    test('check if ChatMessage JSON serialization works', () {
-      final message =
-          ChatMessage('1', chatUserOne, 'Hello, how are you?', chatUserTwo);
-
-      final messageJson = message.toJson();
-
-      final messageFromJson = ChatMessage.fromJson(messageJson);
-
-      expect(messageFromJson.id, message.id);
-      expect(messageFromJson.sender!.firstName, message.sender!.firstName);
-      expect(messageFromJson.receiver!.id, message.receiver!.id);
-      expect(messageFromJson.messageContent, message.messageContent);
-    });
-  });
-}
```

Comment:
Why you have deleted this file?

Reply:
I think i did it by mistake
looking into it

Reply:
can we move all file of chat into chats
@noman2002

Reply:
I am not sure why there are two folders. I guess their must be some purpose for it. Let it be for now.

Reply:
ok